internal/cmd/stack/stack.go

```diff
@@ -223,6 +223,27 @@ func Command() *cli.Command {
 				Before:    authenticated.Ensure,
 				ArgsUsage: "COMMAND",
 			},
+			{
+				Name:  "lock",
+				Usage: "Locks a stack for exclusive use.",
+				Flags: []cli.Flag{
+					flagStackID,
+					flagStackLockNote,
+				},
+				Action:    lock,
+				Before:    authenticated.Ensure,
+				ArgsUsage: "NAME",
```

Comment:
I think you actually want to use `ArgsUsage: cmd.EmptyArgsUsage` here. Same with the unlock command (on the grounds they both only use flags).

Reply:
Of course, it's now fixed.